azure-simple/packer/ansible-ei/roles/integrator/templates/carbon-home/conf/datasources/master-datasources.xml.j2

```diff
@@ -0,0 +1,116 @@
+<datasources-configuration xmlns:svns="http://org.wso2.securevault/configuration">
```

Comment:
Doesn't EI 6.6.0 come with the latest configuration file deployment.toml? If that is the case we don't need to parameterize these files.

Reply:
@ThilinaManamgoda seems like this has not been addressed in any of the production grade installation resources (e.g. Ansible, Kubernetes).